### ISSUE TITLE
wk3 may_epic Bug_Test: 1903 (Auto Save), 1905 (RVI), 1969 (tab initialization), 1973 (horizontal scroll)

### ISSUE THREAD

**Issue by anazariz** [lzim/teampsd#1995]
Testing multiple bugs in one release   

- [x] [1903](https://app.zenhub.com/workspaces/team-psd-workspace-5e4ac9bd8d388dfba6d9aef5/issues/lzim/teampsd/1903) - Auto Save
- [x] [1905](https://app.zenhub.com/workspaces/team-psd-workspace-5e4ac9bd8d388dfba6d9aef5/issues/lzim/teampsd/1905) - Engagement Duration vs RVI
- [x] [1969](https://app.zenhub.com/workspaces/team-psd-workspace-5e4ac9bd8d388dfba6d9aef5/board?epics=87591711_1914&repos=129937957,240349631,240347437,87591711&search=1969) - Data UI tab initialization
- [x] [1973](https://app.zenhub.com/workspaces/team-psd-workspace-5e4ac9bd8d388dfba6d9aef5/issues/lzim/teampsd/1973) - horizontal scroll

Please do you testing with 635 that has already been propagated with the changes.

**Comment by lijenn** [comment]
Thanks @anazariz! All the checks for those bugs look great and worked;

Here are 2 additions that I noticed that may need to be fixed as well, I believe these are easy fixes?
FYI @staceypark 

- [ ] **dataMeas tab in Master Data UI file - Can we have the default for the vertical scroller to be at the top?:**
The vertical scroll bar is defaulted all the way to the bottom. Once we pull in the patient-level data, the data appears to be empty in the dataMeas tab, until the user scrolls all the way back up at the top to see the pulled data. 
How it looks now: 
![image](https://user-images.githubusercontent.com/59668647/119067407-de418180-b996-11eb-9b44-9c53120f8f43.png)

**- [ ] Calculation in MMParams in Master Data UI file - Remove calculation here, should be default 0 for all values in the Param tabs in the Master Data UI file:**
There should be no calculations in any of the Param tabs in the Master Data UI file, this could be leftover data from somewhere or a value may have been put there accidently?:
![image](https://user-images.githubusercontent.com/59668647/119051231-c4dd0d00-b977-11eb-86b8-e346896540f0.png)

**Comment by anazariz** [comment]
@lijenn, thx for the feedback. 

1-Hmm...Apparently the resetting (initialization) of the data/pivot tables leaves the cursor where the last line of data was before reset. I'll check to see if there's way to refocus the cell to Cell(1,1) during propagation. That would effectively fix this issue.     
2-Not sure if the stuff I did changed that number. I'll check. If that has to be programmatically initialized, it may take some time and I may have to delay the release.

**Comment by lijenn** [comment]
Update: For calculation in MMParams tab regarding the 0.5 values, we noticed this explanation here:
![image](https://user-images.githubusercontent.com/59668647/119201064-04246000-ba43-11eb-803e-84ab6db77b1d.png)

@anazariz and I are thinking that this should be left as is and does not affect the team data table creation in Session 2 and 3. 

@lijenn will bring this up in Mon. 8am WG leads, otherwise @anazariz will go ahead and push the bugs we did identify to the latest master data ui file (once testing is done) before he is OOO.
FYI: @lzim @staceypark
